pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -234,6 +234,17 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(request reconcile.Request) (rec
 			return reconcile.Result{RequeueAfter: timeout}, nil
 		}
 		removeFinalizerFromCRDInstance(ctx, instance)
+		err = updateCnsFileAccessConfig(ctx, r.client, instance)
+		if err != nil {
+			msg := fmt.Sprintf("failed to update CnsFileAccessConfig instance: %q on namespace: %q. Error: %+v",
```

Comment:
How about we change the error message to say "failed to update finalizer on CnsFileAccessConfig  instance..."?